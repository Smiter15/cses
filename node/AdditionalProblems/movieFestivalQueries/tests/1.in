100 100
15 99
53 83
38 87
82 86
35 40
59 75
65 97
60 89
7 20
57 69
87 95
94 98
77 79
71 95
35 72
28 66
16 70
73 74
40 44
50 51
53 78
70 80
70 81
23 47
46 98
29 62
64 69
11 85
46 95
86 89
46 61
41 80
64 76
6 22
46 90
17 77
62 88
82 96
20 82
58 63
9 73
79 83
20 27
58 60
83 94
61 78
64 95
61 83
53 97
50 69
67 92
64 88
80 94
3 62
37 74
20 26
75 94
62 83
58 69
55 97
80 90
43 86
32 87
49 87
88 100
69 98
95 100
79 82
79 91
79 84
44 66
78 98
43 78
17 86
16 76
51 80
22 99
85 100
14 88
61 98
16 56
91 99
77 90
19 59
64 70
25 29
35 79
84 91
91 97
92 95
62 78
99 100
75 92
34 90
90 97
24 86
79 86
63 71
3 45
40 89
98 100
75 77
79 90
43 70
8 26
70 74
38 58
83 96
15 25
87 99
70 75
54 58
54 87
24 51
22 87
20 35
67 86
13 46
55 61
19 21
96 99
22 70
58 72
77 84
96 99
33 86
65 83
95 96
61 83
75 78
8 29
6 73
32 89
31 77
58 60
92 95
94 98
21 93
78 82
76 84
77 95
51 89
7 72
93 96
29 78
54 82
3 11
82 89
67 74
18 59
17 34
49 92
25 33
30 57
40 99
45 85
57 88
34 59
95 96
39 63
87 91
73 85
56 69
33 50
1 86
33 70
52 97
69 89
93 98
76 78
19 76
41 72
62 78
26 50
88 96
70 98
65 79
73 84
5 64
66 79
8 87
99 100
92 93
4 19
91 92
23 71
5 6
23 84
47 58
61 77
57 65
38 71
19 48
40 43
92 96
54 56
76 96
39 44
66 68
28 69
