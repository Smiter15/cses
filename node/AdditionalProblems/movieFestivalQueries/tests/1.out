1
0
3
3
1
1
2
2
0
1
1
0
6
4
10
1
4
3
1
0
0
6
2
2
0
9
3
0
4
0
1
7
10
6
1
1
1
11
1
2
5
7
7
0
6
5
0
2
1
4
1
8
1
3
10
6
6
3
0
3
0
3
2
2
11
5
8
5
1
0
7
3
2
2
1
6
2
3
6
2
10
1
0
0
0
6
0
9
1
2
1
4
3
0
1
0
5
1
0
5
